app/src/main/java/com/example/calculator/MainActivity.java

```diff
@@ -1,14 +1,251 @@
+import android.graphics.drawable.Drawable;
 import android.os.Bundle;
+import android.view.View;
+import android.widget.Button;
+import android.widget.TextView;
+
+import com.squareup.picasso.Picasso;
+import com.squareup.picasso.Target;
+
+import org.w3c.dom.Text;
+
+public class MainActivity extends AppCompatActivity implements View.OnClickListener {
+
+    TextView ScreenWithSigns;
+    TextView ScreenWithNumbers;
+    TextView title;
+
+    boolean startOperation = false;
+    boolean startOperationSin = false;
+
+    Button buttonOne;
+    Button buttonOTwo;
+    Button buttonThree;
+    Button buttonFour;
+    Button buttonFive;
+    Button buttonSix;
+    Button buttonSeven;
+    Button buttonEight;
+    Button buttonNine;
+    Button buttonZero;
+    Button buttonPlus;
+    Button buttonMinus;
+    Button buttonSplit;
+    Button buttonMultiply;
+    Button buttonPlusMinus;
+    Button buttonC;
+    Button buttonResetOne;
+    Button buttonEqually;
+
+    Calculator calculator;
+
+    private ConstraintLayout constraintLayout;
+    String url = "https://cdn.pixabay.com/photo/2020/04/16/10/16/mountain-5050026_960_720.jpg";
+
 
-public class MainActivity extends AppCompatActivity {
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
+
+        calculator = new Calculator();
+        initView();
+        initButtons();
+        initLayout();
+        ButtonsSetOnClickListener();
+        loadImageToBackGround();
+
+
     }
+
+    public void initView() {
+        ScreenWithSigns = findViewById(R.id.screenWithSigns);
+        ScreenWithNumbers = findViewById(R.id.screenWithNumbers);
+        title = findViewById(R.id.titleTextView);
+    }
+
+    public void initButtons() {
+        buttonOne = findViewById(R.id.buttonOne);
+        buttonOTwo = findViewById(R.id.buttonTwo);
+        buttonThree = findViewById(R.id.buttonThree);
+        buttonFour = findViewById(R.id.buttonFour);
+        buttonFive = findViewById(R.id.buttonFive);
+        buttonSix = findViewById(R.id.buttonSix);
+        buttonSeven = findViewById(R.id.buttonSeven);
+        buttonEight = findViewById(R.id.buttonEight);
+        buttonNine = findViewById(R.id.buttonNine);
+        buttonZero = findViewById(R.id.buttonZero);
+        buttonPlus = findViewById(R.id.buttonPlus);
+        buttonMinus = findViewById(R.id.buttonMinus);
+        buttonSplit = findViewById(R.id.buttonSplit);
+        buttonMultiply = findViewById(R.id.buttonMultiply);
+        buttonPlusMinus = findViewById(R.id.buttonPlusMinus);
+        buttonC = findViewById(R.id.buttonC);
+        buttonResetOne = findViewById(R.id.buttonResetOne);
+        buttonEqually = findViewById(R.id.buttonEqually);
+    }
+
+
+    public void initLayout() {
+        constraintLayout = findViewById(R.id.constraintLayout);
+    }
+
+    public void ButtonsSetOnClickListener() {
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.